operations/cani/Add_A_Blade_To_A_Cabinet_In_SLS.md

```diff
@@ -0,0 +1,25 @@
+# Add A Blade To A Cabinet In SLS Using CANI
+
+Begin by [initializing a session](https://cray-hpe.github.io/cani/latest/commands/cani_alpha_session_init/), which will import data from SLS and create metadata in SLS' `ExtraProperties` field allowing the process to continue.
+
+```text
+ncn# cani alpha session init csm \
+  --csm-keycloak-username username \
+  --csm-keycloak-password password \
+  --csm-api-host api-gw-service-nmn.local # initialize a session by importing SLS data to cani
+```
+
+> Note: SLS data may be unstable if `cani` has never been run against it, in which case `cani` will not continue.  Resolve any of the data integrity errors before running the above command again.  Known issues and solutions can be found [here](https://cray-hpe.github.io/cani/latest/troubleshooting/known_errors/).
+
+[View available blade types and add one](https://cray-hpe.github.io/cani/latest/commands/cani_alpha_add_blade/) using suggested values.
+
+```text
+ncn# cani alpha add blade -L # see supported blade types
+ncn# cani alpha add blade <blade type> --auto # add a blade using recommended values
+```
+
+[Apply](https://cray-hpe.github.io/cani/latest/commands/cani_alpha_session_apply/) the changes, writing the new data directly to SLS.
+
+```text
+ncn# cani alpha session apply # push changes to SLS
+```
```

Comment:
We most likely need to to copy over the non-sls steps into this doc/refractor the common steps/link to the steps in https://github.com/Cray-HPE/docs-csm/blob/0638f927cf5ac02b6e5880a38c78d8ca96b144fe/operations/node_management/Adding_a_Liquid-cooled_blade_to_a_System.md as we are only modifying SLS, and not touching the other services.

Reply:
No SLS tasks are actually in this doc, but I added the cani steps as a prerequisite